invenio_drafts_resources/services/draft.py

```diff
@@ -51,30 +51,28 @@ def _index_draft(self, draft):
 
     def create(self, data, identity):
         """Create a draft and the associated record (new)."""
```

Comment:
Maybe we need to update the docstring as we don't create a new record on the draft creation :)

Reply:
True, done :)